ops/build_release.sh

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+PROJECT_NAME="Blue Button API"
+BUILD_MESSAGE=""
```

Comment:
@rnagle what should go here? Basically supposed to be how to build and deploy this tag

Reply:
I'm not sure, I think we can drop this and references to it.